static/js/pages/certificates/BrowserImport.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FC, useState } from "react";
+import { Col, Row, Tabs } from "@canonical/react-components";
+import CopyButton from "components/CopyButton";
+
+const FIREFOX = "Firefox";
+const CHROME_LINUX = "Chrome (Linux)";
+const CHROME_WINDOWS = "Chrome (Windows)";
+const TABS: string[] = [FIREFOX, CHROME_LINUX, CHROME_WINDOWS];
+
+const BrowserImport: FC = () => {
+  const [activeTab, handleTabChange] = useState(FIREFOX);
+
+  return (
+    <Row>
+      <Col size={8}>
+        <Tabs
+          links={TABS.map((tab) => ({
+            label: tab,
+            active: tab === activeTab,
+            onClick: () => handleTabChange(tab),
+          }))}
+        />
+
+        {activeTab === FIREFOX && (
+          <div tabIndex={0} role="tabpanel" aria-label="firefox">
+            <ul className="p-list--divided">
+              <li className="p-list__item">
+                Paste this link into the address bar:{" "}
+                <code>about:preferences#privacy</code>{" "}
+                <CopyButton
+                  text="about:preferences#privacy"
+                  tooltipPosition="right"
+                />
```

Comment:
This and the other `CopyButton`s could be right-aligned; in addition they could be wrapped inside the `pre` element, just like in the previous step (Trust).